### PR TITLE
chore: Update itwinui-css to 0.59.1

### PR DESCRIPTION
**Pull request by veekeys**
The only thing I am not sure is Table images and the changes.
If anyone could verify that, would be good.

### CODE REVIEW

File: packages/iTwinUI-react/src/core/Table/Table.tsx [hunk 1]
```diff
@@ -694,10 +694,7 @@ export const Table = <
                       { 'iui-sorted': column.isSorted },
                       column.columnClassName,
                     ),
-                    style: {
-                      ...getCellStyle(column, !!state.isTableResizing),
-                      flexWrap: 'unset',
-                    },
+                    style: getCellStyle(column, !!state.isTableResizing),
```
Comment (mayank99) [inline packages/iTwinUI-react/src/core/Table/Table.tsx:697]: In CSS we are adding a new element `<div className='iui-table-header-wrapper'>`. Don't we need that here?

Comment (mayank99) [inline packages/iTwinUI-react/src/core/Table/Table.tsx:697]: In CSS we are adding a new element `<div className='iui-table-header-wrapper'>`. Don't we need that here?

Comment (veekeys) [reply packages/iTwinUI-react/src/core/Table/Table.tsx]: yeah... missed 2 classes actually.. Added

**Comment by bentleyvk** [comment]
If I remember correctly, content moved a bit horizontally to the left because now side borders are in the first and last cells while previously it was in the row.

**Review by mayank99 — APPROVED** [review]

**Pull request merged**